R/model.R

```diff
@@ -1136,19 +1145,21 @@ survey_mf <- function(survey_ids,
 #' @return Calculated prevalence
 #'
 #' @export
-anc_testing_prev_mf <- function(year, anc_testing, naomi_mf) {
+anc_testing_prev_mf <- function(year, anc_testing, naomi_mf, shape) {
```

Comment:
* Re-sort the aggregated `model_input` data set by area_level, area_id, year.

Reply:
This comment still needs to be addressed, but I think it is elsewhere in the code. Will close this conversation and make comment in new review.